requirements-app.txt

```diff
@@ -26,7 +26,6 @@ PyJWT==2.6.0
 pytz==2021.3
 PyYAML==5.4.1
 SQLAlchemy==1.4.37
-sentry-sdk[flask]==1.10.1
```

Comment:
pretty sure we can delete this file entirely since we've moved to Poetry, but we can try that in a different PR.

Reply:
Hmm yeah, we can try in a different PR to isolate some potential side effect. Let me open a new one exclusively for that.